pkg/authorization/apiserver/admission/restrictusers/restrictusers.go

```diff
@@ -118,6 +121,10 @@ func subjectsDelta(elementsToIgnore, elements []rbac.Subject) []rbac.Subject {
 // in the namespace.
 func (q *restrictUsersAdmission) Validate(a admission.Attributes) (err error) {
 
+	if !q.waitForSyncedStore(time.After(timeToWaitForCacheSync)) {
```

Comment:
this should be after the checks that skip this admission plugin

Reply:
right, done